coordinator/src/node.rs

```diff
@@ -42,13 +46,46 @@ pub mod connection;
 /// The leverage used by the coordinator for all trades.
 const COORDINATOR_LEVERAGE: f32 = 1.0;
 
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct NodeSettings {
+    // At times, we want to disallow opening new positions (e.g. before
+    // scheduled upgrade)
+    pub allow_opening_positions: bool,
```

Comment:
yeah, there's an outermost `Settings` that can't contain others directly due to serde schenigans.

I initially wanted to compose the settings with `WalletSettings` and `NodeSettings` internally, but:
- we're leaking our (arbitrary) abstractions 
- we can't just keep an `RwLock` to the internal ones, because of serde :(